### ISSUE TITLE
Refactor openapi.json endpoint service deployment implementation

### ISSUE THREAD

**Issue by aarne33** [pvarki/docker-rasenmaeher-integration#28]
The openapi.json schema is served at: https://pvarki.github.io/docker-rasenmaeher-integration/openapi.json

Currently the implementation is in the github-action > build.yml. To refactor consider following:
- create openapi.json via fastapi without starting whole rmdev composite
- separate these github-action steps into own yml file
- also consider the webhook usage while fastapi is running. Include into webhook fastapi public url, and then e.g. postman-tool would get fastap.json schema.

**Comment by aarne33** [comment]
https://github.com/pvarki/docker-rasenmaeher-integration/issues/28

**Comment by rambo** [comment]
api submodule dockerfile now has target "openapi"

```bash
docker build --ssh default --target openapi -t rasenmaeher_api:amd64-openapi api/
docker run --rm -it --name rasenmaeher_openapijson rasenmaeher_api:amd64-openapi >openapi.json
```

**Comment by aarne33** [comment]
Closed #27 as it was prototype only. Solid PR to follow.

**Comment by aarne33** [comment]
Closed prototype-PR & corresponding branch. https://github.com/pvarki/docker-rasenmaeher-integration/pull/39
New solid / clean PR to follow.
